sycl/test/devicelib/cmath_fp64_test.cpp

```diff
@@ -1,9 +1,8 @@
-// UNSUPPORTED: windows
-// RUN: %clangxx -fsycl -c %s -o %t.o
-// RUN: %clangxx -fsycl %t.o %sycl_libs_dir/libsycl-cmath-fp64.o -o %t.out
+// REQUIRES: cpu, linux
```

Comment:
Remove this line.  The test should work on Linux and Windows now, and `cpu` is not a requirement for it.

Reply:
Fixed.